app/src/main/java/com/iitp/njack/iitp_connect/data/user/User.java

```diff
@@ -33,4 +33,18 @@
     String email;
     String firstName;
     String lastName;
+    String roll;
+
+    public User(
```

Comment:
This is redundant. You don't need to add the constructor. The `@AllArgsConstructor` annotation does it for you.

Reply:
The redundant constructor has been removed.